launch.go

```diff
@@ -88,6 +93,23 @@ func createWorkspace(srcPaths ...string) (Workspace, error) {
 	return w, nil
 }
 
+// checkPR checks to see if the job name is a pull request and returns its number
```

Comment:
Is the job name the best way to know? Can't we get that out of the SCM URL or something?